src/actions/index.js

```diff
@@ -0,0 +1,53 @@
+import bookService from '../services/books'
+
+export async function fetchBooks (dispatch) {
+  dispatch({ type: 'START_LOADER' })
+  try {
+    const books = await bookService.getAll()
+    dispatch({
+      type: 'SET_BOOKS_COLLECTION',
+      books
+    })
+  } catch (error) {
+    console.log(error)
```

Comment:
Instead of doing `console.log` for every error, we should have an `error` action that we can dispatch, and that also corresponds to an `error` reducer. In short we should keep error as a state for the app also so we can show an error to the frontend displayed nicely. For instance if we cannot fetch the books, then we can display the error message from the server/custom message in a `div` like a flash message